source/utils/fs-async.js

```diff
@@ -0,0 +1,9 @@
+var Promise = require('bluebird');
+var fs = Promise.promisifyAll(require("fs"));
+fs.isFileExists = function(file) {
+  return fs.accessAsync(file, fs.F_OK)
```

Comment:
Just FYI, Node 0.10.x will only supported by the Node.js team until October of this year. Node 0.12.x support will also be dropped in December of this year. (See https://raw.githubusercontent.com/nodejs/LTS/master/schedule.png)